us/en/skymiles/styles/styles.css

```diff
@@ -21,6 +21,7 @@
   --color-delta-gray-light: #f6f6f7;
   --color-delta-red: #e51937;
   --color-delta-red-dark: #98012e;
+  --color-delta-border: #dedfe1;
```

Comment:
You might be able to re-use `--color-delta-gray-middle` as it should be visually close

Reply:
fixed in [f1d8070](https://github.com/hlxsites/delta/pull/89/commits/f1d80700d4d0f621c6466bffc47d9c5b6e4f461c)